projects/www/src/app/pages/guide/signals/signal-store/linked-state.md

```diff
@@ -14,7 +14,7 @@ As a result, the linked state slice is updated automatically whenever any of its
 <ngrx-code-example header="options-store.ts">
 
 ```ts
-import { patchState, signalStore, withLinkedState, withState } from '@ngrx/signals';
+import { patchState, signalStore, withLinkedState, withState, withMethods } from '@ngrx/signals';
```

Comment:
the line is too long, to avoid horizontal scroll:

```suggestion
import {
  patchState,
  signalStore,
  withLinkedState,
  withMethods,
  withState,
} from '@ngrx/signals';
```